scripts/verify-replace.sh

```diff
@@ -5,7 +5,10 @@ GH_BASE_URL_KS=https://github.com/kubesaw/
 GH_BASE_URL_CRT=https://github.com/codeready-toolchain/
 declare -a REPOS=("${GH_BASE_URL_KS}ksctl" "${GH_BASE_URL_CRT}host-operator" "${GH_BASE_URL_CRT}member-operator" "${GH_BASE_URL_CRT}registration-service" "${GH_BASE_URL_CRT}toolchain-e2e" "${GH_BASE_URL_CRT}toolchain-common")
 C_PATH=${PWD}
-ERRORLIST=()
+ERRORREPOLIST=()
+ERRORFILELIST=()
+GOLINTREGEX="[\s\w.\/]*:[0-9]*:[0-9]*:[\s\w]*[:]*[\s\w\">.-]*[\w\s)(*.]*"
```

Comment:
not an expert but should this be something like this
```suggestion
GOLINTREGEX="[\s\w.\/]*:[0-9]*:[0-9]*:[\w\s)(*.\`]*"
```

Reply:
I had tried and tested the regex you suggested, but  we do want the specific format in which the golint displays the error. and i tried to be very specific to the format of golint and hence the above regex i formed. 
for more context, 
`Error: testsupport/wait/member.go:1321:16: G115: integer overflow conversion int -> int32 (gosec)`

`Error: testsupport/wait/awaitility.go:620:9: printf: non-constant format string in call to (*testing.common).Logf (govet)`

`Error: setup/terminal/terminal.go:49:6: the methods of "DefaultTerminal" use pointer receiver and non-pointer receiver. (recvcheck)`

the suggested regex would ideally work , but i thought of being specific to golint format so that we are sure we are not filtering anything else. WDYT ?

Reply:
Okay i have changed the Regex to the one suggested, as i think it would be better to have a smaller regex which works. Thanks

Reply:
Ok, TBH I tested it with in mind this type of syntax only :
```
Error: setup/terminal/terminal.go:49:6: the methods of "DefaultTerminal" use pointer receiver and non-pointer receiver. (recvcheck)
```
I wasn't aware of the other ones. now makes more sense. I think you might be right, my version would fail for the first two.

Reply:
The one which you suggested wont fail, it would just match the line till column number and colon, and get us the whole line during grep ..  i tested it,  it works fine .. so i just switched to it